elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/schedule/JobScheduler.java

```diff
@@ -53,7 +54,7 @@
     private final CoordinatorRegistryCenter regCenter;
     
     private final String elasticJobType;
-    
```

Comment:
Keep indents (including blank lines) consistent with the previous one.
Refer to https://shardingsphere.apache.org/community/en/contribute/code-conduct/

Reply:
thx.